x-pack/plugins/maps/public/classes/sources/es_search_source/es_search_source.tsx

```diff
@@ -589,11 +590,8 @@ export class ESSearchSource extends AbstractESSource implements ITiledSingleLaye
   }
 
   getSourceTooltipContent(sourceDataRequest?: DataRequest): SourceTooltipConfig {
-    const featureCollection: FeatureCollection | null = sourceDataRequest
```

Comment:
Why is this and the condition below getting removed?

Reply:
Since featureCollection is not longer used to determine the number of results, its not needed in the method anymore